### ISSUE TITLE
React Native Web support

### ISSUE THREAD

**Issue by elan** [gitim/react-native-sortable-list#196]
It seems to _almost_ work there, but after a bit of dragging, the drag freezes.

**Comment by nabilfreeman** [comment]
Yeah it's painfully close, but not usable in production. We are implementing a click to move up / down mode as a semi-graceful fallback on RN web
